sdk/src/cameras/itof-camera/camera_itof.cpp

```diff
@@ -441,49 +438,46 @@ aditof::Status CameraItof::setMode(const uint8_t &mode) {
 
     // We want computed frames (Depth & AB). Tell target to initialize depth compute
     if (!m_pcmFrame) {
-        if (!m_ini_depth.empty()) {
-            size_t dataSize = m_depthINIData.size;
-            unsigned char *pData = m_depthINIData.p_data;
+        size_t dataSize = 0;
+        unsigned char *pData = nullptr;
 
-            if (m_depthINIDataMap.size() > 1) {
-                dataSize = m_depthINIDataMap[std::to_string(mode)].size;
-                pData = m_depthINIDataMap[std::to_string(mode)].p_data;
-            }
+        if (m_depthINIDataMap.size() > 1) {
+            dataSize = m_depthINIDataMap[std::to_string(mode)].size;
+            pData = m_depthINIDataMap[std::to_string(mode)].p_data;
+        }
 
-            // Disable the generation of XYZ frames on target
-            std::string s(reinterpret_cast<const char *>(pData), dataSize);
-            std::string::size_type n;
-            n = s.find("xyzEnable=");
-            if (n == std::string::npos) {
-                DLOG(INFO) << "xyzEnable not found in .ini. Can't set it to 0.";
-            } else {
-                s[n + strlen("xyzEnable=")] = '0';
-            }
```

Comment:
These are unnecessary changes. Have you been formatting with clang-format? or something else?